opentutor_classifier_api/tests/test_healthcheck.py

```diff
@@ -81,12 +83,43 @@ def test_503_if_not_healthy(client, message, status):
         assert res.status_code == 503
 
 
-def test_can_override_healthcheck_urls_with_env_var(monkeypatch) -> None:
-    monkeypatch.setenv("HEALTHCHECK_ADMIN", "http://someadmin")
-    assert os.getenv("HEALTHCHECK_ADMIN", "http://wrongurl") == "http://someadmin"
-    monkeypatch.setenv("HEALTHCHECK_HOME", "http://somehome")
-    assert os.getenv("HEALTHCHECK_HOME", "http://wrongurl") == "http://somehome"
-    monkeypatch.setenv("HEALTHCHECK_TUTOR", "http://sometutor")
-    assert os.getenv("HEALTHCHECK_TUTOR", "http://wrongurl") == "http://sometutor"
-    monkeypatch.setenv("HEALTHCHECK_TRAINING", "http://sometraining")
-    assert os.getenv("HEALTHCHECK_TRAINING", "http://wrongurl") == "http://sometraining"
+@pytest.mark.parametrize(
+    "message,status",
+    [("pong!", "success")],
+)
+def test_can_override_healthcheck_admin_url(
+    client, message, status, monkeypatch
+) -> None:
+    with _mock_healthchecks("admin_ok", admin_url="http://someadmin", admin_status=418):
+        monkeypatch.setenv("HEALTHCHECK_ADMIN", "http://someadmin")
+        res = client.get("/classifier/healthcheck/")
+        assert res.json["services"]["admin"]["status"] == 418
+        assert res.status_code == 503
+
+
+@pytest.mark.parametrize(
+    "message,status",
+    [("pong!", "success")],
+)
+def test_can_override_healthcheck_home_url(
+    client, message, status, monkeypatch
+) -> None:
+    with _mock_healthchecks("admin_ok", home_url="http://somehome", home_status=418):
+        monkeypatch.setenv("HEALTHCHECK_HOME", "http://somehome")
+        res = client.get("/classifier/healthcheck/")
+        assert res.json["services"]["home"]["status"] == 418
+        assert res.status_code == 503
+
+
+@pytest.mark.parametrize(
```

Comment:
don't need to parametrize these tests now, just hard code since they're single purpose